projects/kit/src/lib/masks/number/utils/validate-pseudo-separators.ts

```diff
@@ -0,0 +1,17 @@
+import {DEFAULT_PSEUDO_SEPARATORS} from '../../../constants';
+
+export function validatePseudoSeparators({
+    decimalSeparator,
+    thousandSeparator,
+    decimalPseudoSeparators = DEFAULT_PSEUDO_SEPARATORS,
+}: {
+    decimalSeparator: string;
+    thousandSeparator: string;
+    decimalPseudoSeparators?: string[];
+}): string[] {
+    return (
+        decimalPseudoSeparators.filter(
+            char => char !== thousandSeparator && char !== decimalSeparator,
+        ) ?? []
```

Comment:
`decimalPseudoSeparators` is already `string[]`.
`Array.filter` always return array.

```suggestion
        )
```